### ISSUE TITLE
Doubts about inception-resnet-v1 and v2

### ISSUE THREAD

**Issue by qiqiguaitm** [davidsandberg/facenet#449]
@davidsandberg 
Hi, Why is in inception-resnet-v1 and v2 in the project 
the slim.conv2d  automally followed by slim.batch_norm by   the slim.arg_scope   as the following code  in src/models/inception-resnet-v1 and -v2  BUT NO tf.nn.relu.
Why no activation function follow the conv2d and batch_nom. That's strange and unreasonble because nolinearity is dropped.
What's the reason ?

https://github.com/davidsandberg/facenet/blob/master/src/models/inception_resnet_v1.py
def inference(images, keep_probability, phase_train=True,
              bottleneck_layer_size=128, weight_decay=0.0, reuse=None):
    batch_norm_params = {
        # Decay for the moving averages.
        'decay': 0.995,
        # epsilon to prevent 0s in variance.
        'epsilon': 0.001,
        # force in-place updates of mean and variance estimates
        'updates_collections': None,
        # Moving averages ends up in the trainable variables collection
        'variables_collections': [tf.GraphKeys.TRAINABLE_VARIABLES],

with slim.arg_scope([slim.conv2d, slim.fully_connected],
                        weights_initializer=tf.truncated_normal_initializer(stddev=0.1),
                        weights_regularizer=slim.l2_regularizer(weight_decay),
                        normalizer_fn=slim.batch_norm,
                        normalizer_params=batch_norm_params):
        return inception_resnet_v1(images, is_training=phase_train,
                                   dropout_keep_prob=keep_probability, bottleneck_layer_size=bottleneck_layer_size,
                                   reuse=reuse)

Actually in google's  tensorlow/models, the slim.conv2d automally followed by slim.batch_norm and tf.nn.relu.

https://github.com/tensorflow/models/blob/master/slim/nets/inception_resnet_v2.py_
with slim.arg_scope([slim.conv2d, slim.fully_connected],
                      weights_regularizer=slim.l2_regularizer(weight_decay),
                      biases_regularizer=slim.l2_regularizer(weight_decay)):

    batch_norm_params = {
        'decay': batch_norm_decay,
        'epsilon': batch_norm_epsilon,
    }
    # Set activation_fn and parameters for batch_norm.
    with slim.arg_scope([slim.conv2d], activation_fn=tf.nn.relu,
                        normalizer_fn=slim.batch_norm,
                        normalizer_params=batch_norm_params) as scope:
      return scope 

Why you do it so or Do I miss something?

**Comment by ugtony** [comment]
The default activation function of slim.conv2d would be tf.nn.relu, so there is no need to explicitly specify it in the arg_scope unless you want to try other activation functions.

**Comment by qiqiguaitm** [comment]
oh,yes
